### PR TITLE
feat(node-version): ensure node version compatability across contribu…

### PR DESCRIPTION
**Pull request by ThimDeveloper**
# Summary 💭 
To ensure the best compatibility for multiple contributors it is preferable to "soft-lock" the node version to be used for this project. I've opted in for the latest LTS node version 16.14.0 which also is compatible with Next.js.

For all using `direnv` a small check in a `.envrc`-file has been added to automatically use the defined node version when navigating into the directory of the project.

### CODE REVIEW

**Review by willeeklund — APPROVED** [review]
👍

**Comment by ThimDeveloper** [comment]
Hey 👋  @willeeklund .
As I do not have write access to this repository I cannot merge the PRs. Is that something that is handled by the maintainers? The reason I'm asking is that I might need to do some rebasing or similar of my other PRs depending on which merge strategy this repo is using :D

**Pull request merged**
